app/src/main/java/com/mborowiec/antennapod/activity/MainActivity.java

```diff
@@ -509,4 +530,42 @@ public Snackbar showSnackbarAbovePlayer(CharSequence text, int duration) {
     public Snackbar showSnackbarAbovePlayer(int text, int duration) {
         return showSnackbarAbovePlayer(getResources().getText(text), duration);
     }
+
+    /**
+     * Handles the incoming deep link and loads the relevant fragment.
+     * By default, opens the "Add Podcast" feature.
+     *
+     * @param uri incoming deep link
+     */
+    private void handleDeeplink(Uri uri) {
+       if (uri == null) {
+           loadFragment(AddFeedFragment.TAG, null);
+           return;
+       }
+
+        String feature = uri.getQueryParameter("featureName");
+        if(feature != null) {
```

Comment:
Same as above: consider reducing the nesting level, e.g., like

if (feature == null) {
   return;
}

// remaining code